iFixit/multi-user-dev-server
# webpack-multi-user-dev-server: generalize and open source

Summary
---

Take our webpack dev server setup and do these things:
 * Generalize it to take a `usernameToConfigPath` parameter, rather than hardcoding our config paths.
 * Put in a separate, open source npm module.

CR notes
---

Compare this to `webpack.dev-server.js` in our main repo to see the changes.

cr_req 1

QA notes
---

Follow the "example" instructions in `README.md` and make sure it works.

```
git clone (this repo)

npm install
npm start

# in web browser
open localhost:8080/user1/bundle.js
open localhost:8080/user2/bundle.js
# should give you a js file that has a "console.log" at the bottom.
```

Closes https://github.com/iFixit/ifixit/issues/20128
